cogs/rockpaperscissors.py

```diff
@@ -0,0 +1,30 @@
+import nextcord
+from nextcord.ext import commands
+import random
+
+class rps(commands.Cog):
+    @commands.command(
+		name="rps",
+		description="Play Rock Paper Scissors",
+		usage=".rps <option (r, p, s)>",
+		aliases=['rockpaperscissors']
+	)
+	async def rps_command(self, ctx, *, message):
```

Comment:
the `*` is unnecessary here
```suggestion
	async def rps_command(self, ctx, message):
```